cmd/common.go

```diff
@@ -1,11 +1,10 @@
 package cmd
 
 import (
+	"github.com/spf13/cobra"
```

Comment:
Although gofmt doesn't complain, this should be placed with the other github.com/... imports below.